x11-wm/2bwm/2bwm-9999.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit savedconfig
+
+if [[ ${PV} == 9999* ]] ; then
+		inherit git-r3
+		EGIT_REPO_URI="https://github.com/venam/2bwm.git"
+else
+		SRC_URI="https://github.com/venam/2bwm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+		KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="A fast, floating window manager"
+HOMEPAGE="https://github.com/venam/2bwm"
+
+LICENSE="ISC"
+SLOT="0"
+
+DEPEND="
```

Comment:
It seems, that you are missing dependencies.

My compiled package uses different. Could you recheck on your side?

```
  NEEDED               libxcb.so.1 (x11-libs/libxcb)
  NEEDED               libxcb-randr.so.0 (x11-libs/libxcb)
  NEEDED               libxcb-keysyms.so.1 (x11-libs/xcb-util-keysyms)
  NEEDED               libxcb-icccm.so.4 (x11-libs/xcb-util-wm)
  NEEDED               libxcb-ewmh.so.2 (x11-libs/xcb-util-wm)
  NEEDED               libxcb-xrm.so.0 (x11-libs/xcb-util-xrm)
```